README.md

```diff
@@ -52,8 +57,12 @@ The tests can be run by executing `make test`. Any test failing will indicate th
 ## Generating RP reports
 RP reports can be generated by running the following command from inside the virtualenv:
 ```bash
+export AWS_PROFILE=<aws-profile-name>
```

Comment:
This might be already set, if you are using a `.env` file.  Maybe just add a comment on the end, i.e. make this line be:

```
export AWS_PROFILE=<aws-profile-name>  # if not already set via `.env` etc
```